.gitignore

```diff
@@ -23,6 +23,7 @@ node_modules/
 # VSCode build artifacts
 dist/
 .vscode-test/
+.vscode-test-web/
```

Comment:
Is this generated when running `npx @vscode/test-web`?

Reply:
Yes!